demos/universal-sentence-encoder/README.md

```diff
@@ -114,7 +117,7 @@ docker run -it -p 8500:8500 -p 8501:8501 -v $(pwd)/universal-sentence-encoder-mu
 
 Run the client
 ```bash
-python send_strings.py --grpc_port 8500 --input_name inputs --output_name outputs --string "I enjoy taking long walks along the beach with my dog."
+python model_server/demos/universal-sentence-encoder/send_strings.py --grpc_port 8500 --input_name inputs --output_name outputs --string "I enjoy taking long walks along the beach with my dog."
```

Comment:
can we change grpc_port number to 9500? To keep convention that grpc ports start with 9
```suggestion
python model_server/demos/universal-sentence-encoder/send_strings.py --grpc_port 9500 --input_name inputs --output_name outputs --string "I enjoy taking long walks along the beach with my dog."
```